src/components/Discover/TrendingTokens.tsx

```diff
@@ -590,17 +591,18 @@ function TrendingTokenData() {
 
   return (
     <FlatList
-      style={{ marginHorizontal: -20 }}
-      contentContainerStyle={{ paddingHorizontal: 20 }}
+      style={{ marginHorizontal: -20, paddingVertical: 12, marginVertical: -12 }}
```

Comment:
Is this just to expand the list's touchable area?

Reply:
yeah adds a little padding on the top/bottom of the list